csv-bulk-signature/src/main.ts

```diff
@@ -39,7 +39,7 @@ interface TemplateData {
 }
 
 const skippedRows: string[] = [];
-let counter = 0;
+const processedRows: string[] = [];
```

Comment:
added explicit array for signatures processed successfully to access data for status report